master/docs/relnotes/index.rst

```diff
@@ -10,6 +10,57 @@ Release Notes
 
 .. towncrier release notes start
 
+Buildbot ``0.9.4`` ( ``2017-02-08`` )
+=====================================
+
+Database upgrade
+----------------
+
+A database upgrade is necessary for this release (see :bb:cmdline:`upgrade-master`).
+
+Bug fixes
+---------
+
+- Like for ``buildbot start``, ``buildbot upgrade-master`` will now erase an old pidfile if the process is not live anymore instead of just failing.
+- Change properties 'value' changed from String(1024) to Text. Requires upgrade master. (:bug:`3197`)
+- When using REST API, it is now possible to filter and sort in descending order at the same time.
+- Fix issue with :bb:reporter:`HttpStatusPush` raising ``datetime is not JSON serializable`` error.
+- Fix issue with log viewer not properly rendering color codes.
+- Fixed log viewer selection and copy-paste for Firefox (:bug:`3662`).
+- Fix issue with ``DelayedCalled`` already called, and worker missing notification email never received.
+- :bb:cfg:`schedulers`. and :bb:cfg:`change_source` are now properly taking configuration change in account with ``buildbot reconfig``.
```

Comment:
@Frodox you can do this, actually, in a few clicks in the GitHub web interface: open [index.rst](https://github.com/buildbot/buildbot/blob/master/master/docs/relnotes/index.rst), click edit icon, remove extra dot, write commit message, "commit", "create pull request" :)